E2E/assertions/assertions.ts

```diff
@@ -0,0 +1,40 @@
+const { I, game } = inject();
+
+export = {
+    async checkIfTheLinkWasCopied (currentUrl) {
+      I.usePlaywrightTo('check clipboard', async ({ page, browserContext }) => {
+        await browserContext.grantPermissions(['clipboard-read']);
+        const clipboardText = await page.evaluate("navigator.clipboard.readText()");
+        I.assertEqual(currentUrl, clipboardText);
+      });
+  },
+
+  async checkIfTheSPIsRoundedCorrectlyForTheFirstPlayer() {
```

Comment:
why is this extracted? If I understand correctly, this method is only valid when some specific conditions are met (e.g. placing some predefined figures for the first player). If it is true, then this method should be next to the steps that places the system into this expected state, otherwise it only overcomplicates things when you move it into a separate file.